src/transformers/tokenization_fsmt.py

```diff
@@ -37,8 +37,13 @@
     "merges_file": "merges.txt",
 }
 
-PRETRAINED_VOCAB_FILES_MAP = {}
-PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES = {}
+PRETRAINED_VOCAB_FILES_MAP = {
```

Comment:
This is a problem. 

By defining this hardcoded entry the `transformers` stops looking up `tokenizer_config.json`, e.g. as in
https://cdn.huggingface.co/stas/tiny-wmt19-en-de/tokenizer_config.json and everything fails, as it's required unless you specify these values manually when you init the tokenizer.

Reply:
If I revert this, `examples/seq2seq/test_seq2seq_examples.py::test_finetune[stas/tiny-wmt19-en-de]` works again.

Reply:
but this change would break any code relaying on online fsmt model. I should probably add a tiny model test to `tokenizer_fsmt.py`.

Reply:
Oh yes should also add the associated hardcoded config.